Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook

```diff
@@ -74,8 +74,10 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "AMWMetricsUsageDetails\n| summarize c=count(),max(TimeGenerated)\n| project label=iff(c>0,\"Insights as of:\",\"No insights found\"),max_TimeGenerated\n",
+              "query": "AMWMetricsUsageDetails\n| summarize c=count(),max(StartTime)\n| project label=iff(c>0,\"Insights as of:\",\"\"),max_StartTime\n| where label!=\"\"",

```

Comment:
nonblocking nit - `where isnotempty(label)` to also handle null (and IMO is ever so slightly cleaner to read)